src/SFML/Audio/SoundFileWriterOgg.cpp

```diff
@@ -63,7 +63,8 @@ bool SoundFileWriterOgg::open(const std::filesystem::path& filename, unsigned in
     m_channelCount = channelCount;
 
     // Initialize the ogg/vorbis stream
-    ogg_stream_init(&m_ogg, std::rand());
+    static std::mt19937 rng(std::random_device{}());
+    ogg_stream_init(&m_ogg, std::uniform_int_distribution(0, RAND_MAX)(rng));
```

Comment:
I would use `{}` here as it makes it more obvious that the first set of parentheses is initialization and the second is a function call.

Reply:
I didn't do that because it's not precedent in SFML